src/core/unif.ml

```diff
@@ -33,6 +33,30 @@ let pp_problem oc p =
 let no_problems : problems =
   {to_solve  = []; unsolved = []; recompute = false}
 
+
+(** Definition of symbols used to encode unification rules. *)
+module Hint = struct
```

Comment:
I would rather call this module Sym instead. But is it really useful to have a module here?

Reply:
Well, here the module is used only to have an "elegant" name scheme in the caml code, to have `Unif.Hint.atom` rather that `Unif.atom_hint` and `Unif.sign_hint` and alike. So it is purely a question of taste.

Reply:
I see. Ok for Sym then.